olive/systems/python_environment/pass_runner.py

```diff
@@ -21,7 +16,6 @@ def get_args(raw_args):
 
     parser.add_argument("--model_json_path", type=str, required=True, help="Path to input model json file")
     parser.add_argument("--pass_json_path", type=str, required=True, help="Path to pass json file")
-    parser.add_argument("--point_json_path", type=str, default=None, help="Path to point json file")
```

Comment:
I am not very familar with this part of logics. QQ: what is the point_json_path? Is that safe to remove it? If so, please also update in PR description.

Reply:
Move the config update to consistent with that one in AzureML system